packages/fast-usdc/src/fast-usdc.start.js

```diff
@@ -206,13 +214,23 @@ export const startFastUSDC = async (
   await publishFeedPolicy(storageNode, feedPolicy);
 
   const {
-    issuers: { PoolShares: shareIssuer },
+    issuers: fastUsdcIssuers,
     brands: { PoolShares: shareBrand },
   } = await E(zoe).getTerms(instance);
+  /** @type {{ PoolShares: Issuer<'nat'> }} */
+  // @ts-expect-error see zcf.makeZCFMint(...) in fast-usdc.contract.js
+  const { PoolShares: shareIssuer } = fastUsdcIssuers;
   produceShareIssuer.resolve(shareIssuer);
   produceShareBrand.resolve(shareBrand);
   await publishDisplayInfo(shareBrand, { board, chainStorage });
 
+  const { denom, issuerName } = ShareAssetInfo;
+  trace('addAsset', denom, shareBrand);
+  await E(bankManager).addAsset(denom, issuerName, issuerName, {
```

Comment:
not objecting, but why use a ZCFMint when the start eval is now concerned with the Issuer?

Reply:
yes. A ZCFMint is the normal Zoe API for a token issued by a contract.